ENDPOINT_STATUS.md

```diff
@@ -1,32 +1,35 @@
 # Status
 
-Check here to see if the library is known to be working or not against each DiGA endpoint, 
+Check here to see if the library is known to be working or not against each DiGA endpoint,
 together with information on how many companies are using each endpoint, and issue tracking if the endpoint is not working:
 
-- CODE TEST OK = test codes are working
-- CODE TEST NOT OK = test codes are not working
-- CODE OK = real code validation has been attemped and verified to work
-- CODE NOT VERIFIED = real code validation probably works, but it has not been tested on a real code yet
-- CODE NOT OK = real code validation does not work
-- BILLING TEST OK = test bill is working (the APIs dont support this completely, but as far as you can get it is working)
-- BILLING TEST NOT OK = test bill is not working
-- BILLING OK = billing has been attempted and verified to work
-- BILLING NOT VERIFIED = billing probably works, but it has not been verified with a real bill yet
-- BILLING NOT OK = billing does not work
-
+| Test Result          | Explanation                                                                                           |
```

Comment:
Could it make sense to extract the patch for ENDPOINT_STATUS.md into another MR? Seems like it can be merged in to main independently from changes in the two other files.

Reply:
yes @bokchan that would be nice :)

Reply:
Ok. You wanna have the honor @SebastianRehfeldt ? :)

Reply:
Sure thing: https://github.com/alex-therapeutics/diga-api-client/pull/59